src/resources/filters/layout/meta.lua

```diff
@@ -28,7 +28,7 @@ function layoutMetaInject()
       local kCodeBlockBackground = 'code-block-bg'
 
       -- Track whether to show a border or background
-      local useCodeBlockBorder = (adaptiveTextHighlighting and meta[kCodeBlockBorderLeft] == nil and meta[kCodeBlockBackground] == nil) or meta[kCodeBlockBorderLeft] ~= nil
+      local useCodeBlockBorder = (adaptiveTextHighlighting and meta[kCodeBlockBorderLeft] == nil and meta[kCodeBlockBackground] == nil) or not meta[kCodeBlockBorderLeft] == false
       local useCodeBlockBg = meta[kCodeBlockBackground] ~= nil
```

Comment:
I think we should also be checking for `~= false` here as well (since currently this has exactly the same bug!)

Reply:
I wasn't sure if I understood the features here correctly and if I found a confirm bug. Should have been more confident. 

This is the confirmation I was looking for!